README.md

```diff
@@ -196,7 +196,8 @@ Factory function to that creates a read-only or writeable client. By providing a
 // read-only client that has access to all read APIs
 const arIOReadable = ArIO.init()
 
-const browserSigner = new ArConnectSigner(window.arweaveWallet);
+const arweave = Arweave.init({ ...arweave api config })
```

Comment:
simplify this example

```suggestion
const arweave = Arweave.init({})
```

Reply:
doing this has side affects with arweave-js - you cool with an ar-io.dev config in the example?

Reply:
what do you mean? where are arweave, api and config coming from in this example? not clear

Reply:
it was a description of what to put - changing to this: 
```
const arweave = Arweave.init({
  host: 'ar-io.dev',
  port: 443,
  protocol: 'https'
})
```